lms/djangoapps/instructor/tests/test_proctoring.py

```diff
@@ -36,24 +34,74 @@ def setUp(self):
         self.instructor = AdminFactory.create()
         self.client.login(username=self.instructor.username, password="test")
 
-        CourseFinanceAdminRole(self.course.id).add_users(self.instructor)
+    def setUpCourseUrl(self, course):
+        """
+        Create URL for instructor dashboard
+        """
+        self.url = reverse('instructor_dashboard', kwargs={'course_id': course.id.to_deprecated_string()})
 
-    def test_pass_proctoring_tab_in_instructor_dashboard(self):
+    def setUpCourse(self, enable_proctored_exams, enable_timed_exams):
         """
-        Test Pass Proctoring Tab is in the Instructor Dashboard
+        Create course based on proctored exams and timed exams values
         """
+        self.course = CourseFactory.create(enable_proctored_exams=enable_proctored_exams,
+                                           enable_timed_exams=enable_timed_exams)
+        self.setUpCourseUrl(self.course)
+
+    @ddt.data(
+        (True, False),
+        (False, True)
+    )
+    @ddt.unpack
+    def test_proctoring_tab_visible_for_global_staff(self, enable_proctored_exams, enable_timed_exams):
+        """
+        Test Proctoring Tab is visible in the Instructor Dashboard
+        for global staff
+        """
+        self.setUpCourse(enable_proctored_exams, enable_timed_exams)
+
         self.instructor.is_staff = True
         self.instructor.save()
 
-        response = self.client.get(self.url)
-        self.assertIn(self.proctoring_link, response.content)
-        self.assertIn('Allowance Section', response.content)
+        # expects that proctoring tab is visible for global staff
```

Comment:
sound more readable as 'verify that proctoring tab is visible for global staff'. It is good as it is. Just a suggestion.
If you think this can be with 'verify'. Please change at other places too.

Reply:
sure